lnwallet/channel.go

```diff
@@ -2598,6 +2728,10 @@ func (lc *LightningChannel) createCommitDiff(
 			copy(htlc.OnionBlob[:], pd.OnionBlob)
 			logUpdate.UpdateMsg = htlc
 
+			logUpdates = append(logUpdates, logUpdate)
```

Comment:
Why the short circuit here?

Reply:
Was put in to avoid double adding the udpate to `logUpdates` below. I think this needs to be reworked a little, because I think we aren't add `OpenCircuitRef`s for Adds

Reply:
Fixed